Src/libCZI/libCZI_Write.h

```diff
@@ -440,11 +440,15 @@ namespace libCZI
         /// \return The "pre-filled" metadata object if successful.
         virtual std::shared_ptr<libCZI::ICziMetadataBuilder> GetPreparedMetadata(const PrepareMetadataInfo& info) = 0;
 
-        /// Finalizes the CZI (ie. writes out the final directory-segments) and closes the file.
+        /// Finalizes the CZI (i.e. writes out the final directory-segments) and closes the file.
         /// Note that this method must be called explicitely in order to get a valid CZI - calling the destructor alone will
         /// close the file immediately without finalization.
         virtual void Close() = 0;
 
+        /// Gets the statistics about the sub-blocks. This statistics is aggregated from the subblocks as they are added.
```

Comment:
Nitpicky: mix of "sub-blocks" and "subblocks" with and without dash. Irrelevant really, just in case you care :)

Reply:
right, that's bad - however, I can live with that for now :-)